gc/gc-base/src/main/java/org/projectnessie/gc/base/IdentifyContentsPerExecutor.java

```diff
@@ -241,30 +245,32 @@ private scala.collection.Iterator<Row> getExpiredContentRows(
               api.close();
             });
     return references.flatMap(
-        reference ->
-            JavaConverters.asScalaIterator(
-                    walkAllCommitsInReference(
-                        api,
-                        GCUtil.deserializeReference(reference),
-                        liveContentsBloomFilterMap,
-                        runId,
-                        startedAt))
-                .toTraversable());
+        reference -> {
+          Reference ref = GCUtil.deserializeReference(reference);
```

Comment:
This is a noop change that unnecessarily increases the amount of changes and review effort